--- htpdate-1.1.3/htpdate.c.orig	2015-12-21 04:27:41.000000000 -0600
+++ htpdate-1.1.3/htpdate.c	2016-02-03 16:01:49.888966935 -0600
@@ -366,7 +366,7 @@ static int htpdate_adjtimex( double drif
 
 	/* Read current kernel frequency */
 	tmx.modes = 0;
-	ntp_adjtime(&tmx);
+	adjtimex(&tmx);
 
 	/* Calculate new frequency */
 	freq = (long)(65536e6 * drift);
@@ -381,7 +381,7 @@ static int htpdate_adjtimex( double drif
 
 	/* Become root */
 	swuid(0);
-	return( ntp_adjtime(&tmx) );
+	return( adjtimex(&tmx) );
 
 }
 
